src/sonic-host-services/.gitignore

```diff
@@ -14,3 +14,4 @@ dist/
 .coverage
 coverage.xml
 htmlcov/
+/.pytest_cache/
```

Comment:
Leading slash should not be necessary; also keep list alphabetized:

```
.coverage
.pytest_cache/
coverage.xml
htmlcov/
```

Reply:
That was automatically added by the tool I am using.

Reply:
What tool is that?

Reply:
I think it is `Git` plugin in `Eclipse`